src/App.tsx

```diff
@@ -20,16 +37,18 @@ export class App extends React.Component<{}, State> {
 
               <div className="control">
                 <input
+                  name="query"
                   type="text"
+                  value={query}
+                  onChange={this.handleChange}
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
                 />
               </div>
             </div>
           </div>
-
-          <MoviesList movies={moviesFromServer} />
+          <MoviesList movies={visibleMovies} />
```

Comment:
And here you can use this method
```suggestion
          <MoviesList movies={this.getVisibleMovies()} />
```